models/orderDao.js

```diff
@@ -0,0 +1,150 @@
+const appDataSource = require('./dataSource');
+
+const createOrder = async (paymentData, userId) => {
+  const { aid, tid, payment_method_type: paymentType, quantity } = paymentData;
+  const {
+    total: totalAmount,
+    tax_free: taxFree,
+    vat,
+    point,
+    discount,
+  } = paymentData.amount;
+
+  const queryRunner = appDataSource.createQueryRunner();
+  await queryRunner.connect();
+  await queryRunner.startTransaction();
+
+  try {
+    const [ticketInfo] = await queryRunner.query(
+      `
+      SELECT
+        user_id,
+        JSON_ARRAYAGG(time_tables_theater_seat_id) AS seatList
+      FROM ticketings
+      WHERE user_id=?
+      GROUP BY user_id
+      `,
+      [userId]
+    );
+    const orderStatus = Object.freeze({
+      standBy: 1,
+      completed: 2,
+      canceled: 3,
+    });
+
+    await queryRunner.query(
+      `
+      INSERT INTO orders (
+        user_id,
+        tid,
+        quantity,
+        total_amount,
+        tax_free_amount,
+        order_status_id
+      )
+      VALUES (
+        ?,
+        ?,
+        ?,
+        ?,
+        ?,
+        ?
+      )
+      `,
+      [userId, tid, quantity, totalAmount, taxFree, orderStatus.completed]
+    );
+
+    const [order] = await queryRunner.query(
+      `
+      SELECT
+        id,
+        user_id
+      FROM orders
+      WHERE user_id=?
+      `,
+      [userId]
+    );
+
+    const tickets = [];
```

Comment:
```suggestion
    let tickets = [];
```
array를 순회하면서 push 할거라면 let으로 선언해주세요~